.vscode/c_cpp_properties.json

```diff
@@ -12,7 +12,8 @@
             ],
             "cStandard": "c17",
```

Comment:
Not sure if this can be changed to use Visual Studio 2015, because Noita for some reason used it. I am using it for LuaRocks. [source: CTRL+F: vcvars_ver=14.0](https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170#vcvarsall-syntax).
Not sure about this, maybe you know better or can have a reading.

Reply:
Nevermind, looks like it's the cpp standard.